Automattic/vip-go-mu-plugins
# bump elasticpress

## Description

see https://github.com/Automattic/ElasticPress/pull/83 for more details.

## Changelog Description

### Plugin Updated: Search

Update ElasticPress to avoid warnings due to deprecated operation in ElasticSearch 7.X and higher.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
We only use multi-get in validate contents so:
So running lando wp vip-search health validate-contents before the change will throw a warning, and will work as expected without warning after the change.
